examples/spice.mzLib.txt

```diff
@@ -0,0 +1,649 @@
+<mzSpecLib 1.0>
+MS:1003188|library name=spice
+MS:1003190|library version=1.0
+MS:1001017|release date=7/7/2023 7:06:42 PM
+MS:1003203|constituent spectrum file=mzspec:USI000000:spice_ph/cal_10ng.raw

```

Comment:
I _think_ this might need to be in the `Spectrum` `all` attribute set.

Reply:
Because I have added the scan number, it was moved to the spectrum section for each entry.